extensions/parquet/table/src/test/java/io/deephaven/parquet/table/ParquetTableFilterTest.java

```diff
@@ -466,6 +535,18 @@ public void partitionedNoDataIndexTest() {
         filterAndVerifyResults(diskTable, memTable, "symbol < `s100`");
         filterAndVerifyResults(diskTable, memTable, "symbol = `s500`");
 
+        // Conditional on partition column
+        filterAndVerifyResults(diskTable, memTable, "symbol = `s` + `500`");
```

Comment:
I don't think we are actually creating something that is serial here.  The values are immutable.  You should verify that in a debugger.  I do wonder if a stateless filter is worth testing on partitions just for funzies.  We could use a counting where filter (or io.deephaven.engine.testutil.filters.RowSetCapturingFilter) to be sure.

Reply:
The lines you flagged won't produce a serial filter, but they do trigger the `permitParallelization() == false` case which is still an interesting case.

I'll add some stateless filter cases to `partitionedDataSerialFilterTest` to verify filter call counts.